CONTRIBUTING.md

```diff
@@ -1,11 +1,11 @@
-## Contributing
+## Contributing to the _GitHub Extension for Visual Studio_
```

Comment:
Yeah, I totally understand the logic. In the case of people filing issues, however, they're not going to see anything rendered, since the issue template is dropped into the issue body - and users that can't be bothered to read the project description itself are highly likely not to even bother looking at the preview to see if what they're writing makes actual sense.